web3.js/src/message.ts

```diff
@@ -73,7 +80,16 @@ export class Message {
   constructor(args: MessageArgs) {
     this.header = args.header;
     this.accountKeys = args.accountKeys.map(account => new PublicKey(account));
-    this.recentBlockhash = args.recentBlockhash;
+    if (
+      // eslint-disable-next-line no-prototype-builtins
```

Comment:
Oh, sorry if I confused you here. I saw what you wrote about `Object.prototype.hasOwnProperty.call(args, 'latestBlockhash')` and meant that I learned something new, not that you should change it back and suppress the lint warning.

Reply:
No worries. I couldn't find your comment about this. I can put this back to the way it was before :)